vendors/espressif/boards/esp32/aws_demos/config_files/aws_wifi_config.h

```diff
@@ -59,7 +59,8 @@
 /**
  * @brief Max passphrase length
  */
-#define wificonfigMAX_PASSPHRASE_LEN          ( 64 )
+// Maximum allowed WPA2 passphrase length (per specification) is 63
```

Comment:
Good note. Could you please reformat into existing comment Line#59 ?
```suggestion
/**
 * @brief Max passphrase length
 * Maximum allowed WPA2 passphrase length (per specification) is 63
 */
```